public/components/trace_analytics/components/services/services_content.tsx

```diff
@@ -42,6 +45,8 @@ export function ServicesContent(props: ServicesProps) {
     jaegerIndicesExist,
   } = props;
   const [tableItems, setTableItems] = useState([]);
+
+  const [trigger, setTrigger] = useState<'open' | 'closed' | undefined>('closed');
```

Comment:
Do we ever use the `undefined` value?

Reply:
No, thanks for catching that - was fixed